node-server/src/database/__mocks__/node-dao.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+
+const testData = require('../../utils/test-data')
+
+class NodeDao {
+  nodes
+
+  // eslint-disable-next-line no-unused-vars
+  constructor(querier) {
+    console.log('Mock database table Node created')
+  }
+
+  createTableNode = () => {
+    this.nodes = testData.nodes
+  }
+
+  getNode = () => {
+    return Promise.resolve(this.nodes[0])
+  }
+
+  addNewNode = (node) => {
+    this.nodes = [node]
```

Comment:
This overwrites all the previous nodes with the single added node. Should probably be [...nodes, node] ?